test/suites/storage_driver_zfs.sh

```diff
@@ -374,6 +374,70 @@ do_storage_driver_zfs() {
   lxc delete -f c8
   lxc storage unset lxdtest-"$(basename "${LXD_DIR}")" volume.zfs.remove_snapshots
 
+  # Test recursive copy snapshot cleanup.
+  # Create separate storage pool for the test.
+  lxc storage create test-pool zfs
+
+  # Create the first container
+  lxc init testimage t1 --storage test-pool
+
+  # Make two copies.
+  lxc copy t1 t2
+  lxc copy t1 t3
+
+  # Verify two copy snapshots exist.
+  copy_snaps=$(zfs list -t snapshot -H -o name test-pool/containers/t1)
+  copy_snap_count=$(echo "$copy_snaps" | grep -c "@copy-")
+  [ "$copy_snap_count" -eq 2 ]
```

Comment:
This feels more direct.
```suggestion
  [ "$(zfs list -t snapshot -H -o name test-pool/containers/t1 | grep -cF "@copy-")" -eq 2 ]
```